pyopenstates.py

```diff
@@ -25,14 +25,17 @@
 
 __version__ = "1.2.0"
 
-API_ROOT = "https://openstates.org/api/v1/"
+API_ROOT = "https://v3.openstates.org"
 DEFAULT_USER_AGENT = "pyopenstates/{0}".format(__version__)
 ENVIRON_API_KEY = os.environ.get('OPENSTATES_API_KEY')
 
 session = Session()
+session.headers.update({"Accept": 'application/json'})
 session.headers.update({"User-Agent": DEFAULT_USER_AGENT})
 if ENVIRON_API_KEY:
     session.headers.update({'X-Api-Key': ENVIRON_API_KEY})
+else:
+    print("Warning: No API Key found, set API_KEY")
```

Comment:
could we make this say OPENSTATES_API_KEY since that's the name of the environment variable